closed/OpenJ9.gmk

```diff
@@ -109,6 +109,19 @@ OPENJ9_MISC_FILES := \
 OPENJ9_NOTICE_FILES := openj9-notices.html
 OPENJ9_REDIRECTOR := redirector/$(LIBRARY_PREFIX)jvm_b156$(SHARED_LIBRARY_SUFFIX)
 
+OPENJ9_DDR_FILES := 
+
+ifeq (true,$(OPENJ9_ENABLE_DDR))
+  $(OUTPUT_ROOT)/vm/j9ddr.dat : run-ddrgen
+  run-ddrgen :
+	$(MAKE) -C $(OUTPUT_ROOT)/vm/ddr -f run_omrddrgen.mk \
+		VERSION_MAJOR=9 \
+		OPENJDK_VERSION_NUMBER_FOUR_POSITIONS=$(VERSION_NUMBER_FOUR_POSITIONS) \
+		CC="$(CC)" CXX="$(CXX)" \
+		$(EXPORT_MSVS_ENV_VARS)
+	OPENJ9_DDR_FILES += j9ddr.dat
```

Comment:
This line cannot start with a tab else it will be considered to be part of the script to produce `run-ddrgen `.

Also, there should be a rule
```
.PHONY : run-ddrgen
```